pkg/util/tracing/span.go

```diff
@@ -277,6 +294,11 @@ func (s *spanInner) SetVerbose(to bool) {
 	}
 }
 
+func (s *spanInner) SetVerboseRecursively(to bool) {
+	s.SetVerbose(to)
```

Comment:
This line is extraneous, setVerboseRecursively already sets it for the span it's being called on.